sdk/iot/README.md

```diff
@@ -59,7 +83,7 @@ int main()
 
 ### Properties
 
-Included in the IoT SDK are helper functions to form and manage properties for IoT Hub services. Implementation starts by using the `az_iot_hub_client_properties_init()` API. The user is free to intitialize using an empty, but appropriately sized, span to later append properties or an already populated span containing a properly formated property buffer. "Properly formatted" properties follow the form `{key}={value}&{key}={value}`.
+Included in the Azure IoT Embedded C SDK are helper functions to form and manage properties for IoT Hub services. Implementation starts by using the `az_iot_hub_client_properties_init()` API. The user is free to intitialize using an empty, but appropriately sized, span to later append properties or an already populated span containing a properly formated property buffer. "Properly formatted" properties follow the form `{key}={value}&{key}={value}`.

```

Comment:
There is no such thing as the "Azure IoT Embedded C SDK". There is the "Azure SDK for Embedded C" which contains some IOT-related client libraries. I think terminology is really important here so that we can and customer can speak the same language when referring to things.